### PR TITLE
new work page

### PR DESCRIPTION
**Pull request by csergiuu**

### CODE REVIEW

File: css/src/ui/_buttons.scss [hunk 1]
```diff
@@ -29,3 +29,43 @@
     color: $red;
   }
 }
+
+
+.default-link{
+  position: relative;
+  display: inline-block;
+  font-size:18px;
```
Comment (joaovpmamede) [inline css/src/ui/_buttons.scss]: spacing!!!
`font-size: 18px;`

File: css/src/ui/_line-chart.scss [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    &:nth-child(2){
+      width: 33%;
+      transition-delay:400ms;
+     
+      &:before{
+        content:'app - 53';
+      }
+      &:after{
+        // background-color: rgba($red, 0.33);
```
Comment (joaovpmamede) [inline css/src/ui/_line-chart.scss]: If not needed remove comments please.

File: css/src/ui/_work-wrap.scss [hunk 3]
```diff
@@ -0,0 +1,148 @@
+.work-wrap{
+	position: relative;
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+
+    &:not(:nth-child(2)){
+		border-top:1px solid rgba($gray, 0.2);
```
Comment (joaovpmamede) [inline css/src/ui/_work-wrap.scss]: Spacing.
Also try to use 2 spaces.

File: js/main.js [hunk 4]
```diff
@@ -85,6 +85,34 @@ MINUS.work = (function($) {
     }
   }
 
+  setInterval(function(){
+    $('.line-chart span').addClass('is-active');
+  }, 500);
+
+
+  var animation_elements = $('.work-wrap');
```
Comment (joaovpmamede) [inline js/main.js]: 🐍 _case should only be used in ruby or php (?!)... we really need to start a "styleguide" and bring some linter rules to our projects.

File: work/_posts/2016-10-19-wellclub.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ client: 'America'
 date: 2016-10-19 15:30
 uri: 'http://therapp.club'
 image:
- - url: '../../images/work/wellclub.jpg'
+ - url: '../../images/work/thumbs/thumb-wellclub.jpg'
```
Comment (miguelm) [inline work/_posts/2016-10-19-wellclub.md]: imagems da app

**Comment by miguelm** [comment]
@csergiuu basically we need app images on:
- redbull
- quill
- wellclub
- crumbs
- origama test with a product page screen

**Comment by miguelm** [comment]
@dcruz @joaovpmamede final round of approval if you can provide feedback

**Pull request merged**